src/main/java/se/sundsvall/parkingpermit/api/ProcessResource.java

```diff
@@ -33,18 +34,18 @@ public class ProcessResource {
 	@Autowired
 	private ProcessService service;
 
-	@PostMapping(path = "start/{businessKey}", produces = { APPLICATION_JSON_VALUE, APPLICATION_PROBLEM_JSON_VALUE })
-	@Operation(description = "Start a new process instance for the provided business key")
+	@PostMapping(path = "start/{caseNumber}", produces = { APPLICATION_JSON_VALUE, APPLICATION_PROBLEM_JSON_VALUE })
+	@Operation(description = "Start a new process instance for the provided caseNumber")
 	@ApiResponse(responseCode = "202", description = "Accepted", content = @Content(schema = @Schema(implementation = StartProcessResponse.class)))
 	@ApiResponse(responseCode = "400", description = "Bad request", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(oneOf = { Problem.class, ConstraintViolationProblem.class })))
 	@ApiResponse(responseCode = "404", description = "Not found", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
 	@ApiResponse(responseCode = "500", description = "Internal Server error", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
 	@ApiResponse(responseCode = "502", description = "Bad Gateway", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
 	public ResponseEntity<StartProcessResponse> startProcess(
-		@Parameter(name = "businessKey") @PathVariable final String businessKey) {
+		@Parameter(name = "caseNumber") @PathVariable final String caseNumber) {
 
-		var startProcessResponse = new StartProcessResponse(service.startProcess(businessKey));
-		LOGGER.info("Request for start of process has been received, resulting in an instance with id {}", startProcessResponse.getProcessId());
+		var startProcessResponse = new StartProcessResponse(service.startProcess(caseNumber));
+		LOGGER.info("Request for start of process for caseNumber {} has been received, resulting in an instance with id {}", caseNumber, startProcessResponse.getProcessId());
 		return new ResponseEntity<>(startProcessResponse, HttpStatus.ACCEPTED);
```

Comment:
Nothing wrong here, just a comment based on my personal preference:

Maybe use ResponseEntity builder instead (for better readability)?
`return accepted().body(startProcessResponse);`